app/pandas.ipynb

```diff
@@ -435,9 +849,7 @@
    "cell_type": "code",
    "execution_count": null,
    "outputs": [],
-   "source": [
-    "# filter s2 here"
-   ],
+   "source": "odds_series = s1[s1 % 2 != 0]",
```

Comment:
The filtering for odd numbers is applied to `s1` instead of `s2`. To filter `s2` for odd numbers, use `s2[s2 % 2 != 0]`.